makeelf/utils.py

```diff
@@ -1,8 +1,17 @@
 #!/usr/bin/env python3
 ## \file utils.py
 #  \brief Utility functions
+
+# Old code was quite slow on large files (e.g., several seconds on a 32MB ELF).
+# Most of this was xor.  Large speedup from this SO answer:
+#https://stackoverflow.com/questions/2119761/simple-python-challenge-fastest-bitwise-xor-on-data-buffers
+#
+# Although, this does add numpy as a dependency, so perhaps this should be checked at runtime?
```

Comment:
This is perfect information for commit message, but serves no purpose inline, as whoever reads this does not see previous version.